code/__DEFINES/antag_defines.dm

```diff
@@ -104,7 +104,7 @@ GLOBAL_LIST(contractors)
 #define ORG_PROB_HUNTER 10
 #define ORG_PROB_MILD 20
 #define ORG_PROB_AVERAGE 60
-#define ORG_PROB_HIJACK 10
+#define ORG_PROB_HIJACK 5
```

Comment:
While I dont believe this is strictly necessary and this will work as is, you should raise the probability of another org type to have the total be 100. I would increase ORG_PROB_AVERAGE to 65. (I coded orgs, for reference)

Reply:
Yeah that makes sense. I should have seen that coming. I just forgot that we added in organizations (I know you coded it and I reviewed it lol whoops...). I have reached out to the teams, and decided to go with your suggestion. Thanks!